pkg/controller/garbagecollector/garbagecollector.go

```diff
@@ -43,6 +44,8 @@ import (
 
 const ResourceResyncTime = 60 * time.Second
 
+const OrphanFinalizerID = "orphanFinalizer"
```

Comment:
Exactly :-)

On Wednesday, May 18, 2016, Daniel Smith <notifications@github.com> wrote:

> In pkg/controller/garbagecollector/garbagecollector.go
> <https://github.com/kubernetes/kubernetes/pull/25599#discussion_r63803135>
> :
>
> > @@ -43,6 +44,8 @@ import (
> >
> >  const ResourceResyncTime = 60 * time.Second
> >
> > +const OrphanFinalizerID = "orphanFinalizer"
>
> Also please put it in a different package so apiserver doesn't have to
> import the garbage collector.
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/25599/files/7b7917ddcb2f2ab0d8c22748df3668346ea69505#r63803135>
>